### PR TITLE
Convert markdown issue templates to forms

### PR DESCRIPTION
**Pull request by gravityfargo**
This PR converts the traditional markdown issue template to the beta forms template. The documentation can be found [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms). 

You can find the live version of what the form looks like for users on my fork and prefilled issues I submitted to show you the result of those forms. 

I also added the discord link as an item in the [template picker page](https://github.com/gravityfargo/liquidctl/issues/new/choose). 

[bug report form](https://github.com/gravityfargo/liquidctl/issues/new?assignees=&labels=bug&template=bug_report.yml)
[example issue](https://github.com/gravityfargo/liquidctl/issues/1)

[new device form](https://github.com/gravityfargo/liquidctl/issues/new?assignees=&labels=new+device&template=new-device.yml)
[example issue](https://github.com/gravityfargo/liquidctl/issues/2)

[other issues form](https://github.com/gravityfargo/liquidctl/issues/new?assignees=&labels=question&template=help-request.yml)
[example issue](https://github.com/gravityfargo/liquidctl/issues/3)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: Report an issue with existing supported devices.
+labels: ["bug"]
+body:
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:4]: The order in which each information is requested was intentional: a good description of the bug is the most important part, followed by the output from all relevant commands, and then the additional information.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  - type: input
+    id: info-device
+    attributes:
+      label: Affected Device
+      description: What is the name of the device you are using that is not functioning as expected? If more than one devices are experiencing issues list them here separated by a comma.
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]: This is unnecessary long, IMO:

```suggestion
      description: Which device isn't functioning as expected?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      placeholder: e.g. Corsair Hydro H100i Pro XT
+    validations:
+      required: true
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:12]: Some bugs aren't specific to any particular device, so I don't think this should be required.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  - type: input
+    id: info-distro
+    attributes:
+      label: Linux Distribution
+      description: Which Linux distribution are you using?
+      placeholder: ex. Arch
+    validations:
+      required: true
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:20]: liquidctl supports platforms other than Linux (even though that happens to be my main/preferred one).

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  - type: input
+    id: info-version
+    attributes:
+      label: Liquidctl Version
+      description: Output of `liquidctl --version`
+      placeholder: e.g. liquidctl v1.9.0 (Linux-5.16.16-arch1-1-x86_64-with-glibc2.35) 
+    validations:
+      required: true
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:36]: Not a major thing, but can you make both descriptions account for the scenario where the bug is in the installation procedure?

(But keeping them short & simple for the far more common case of a bug after the installation was successful).

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    attributes:
+      label: Describe the bug
+      description: Please start by describing the bug in as much detail as possible.
+      placeholder: ex. "liquidctl fails to read the fan speed of the two attached fans"
+    validations:
+      required: true
+  - type: textarea
+    id: user-solutions
+    attributes:
+      label: Command output with debug flag
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]:
```suggestion
      label: Output of all relevant commands (with `--debug` flag)
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 7]
```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Discord
+    url: https://discord.gg/GyCBjQhqCd
+    about: "Ask for support in our Discord Server!."
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/config.yml:5]: I rather not forward users to Discord, which turns into a mess for anything that might might eventually be turned into an issue.

We have discussions enabled in this repository, that's probably a better place to redirect these types of questions.

File: .github/ISSUE_TEMPLATE/help-request.yml [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+description: For all other questions / issues
+labels: ["question"]
+body:
+  - type: textarea
+    id: question-body
+    attributes:
+      label: Describe your question.
+      description: Please provide any relevant information
+    validations:
+      required: true
```
Comment (jonasmalacofilho) [inline .github/ISSUE_TEMPLATE/help-request.yml:11]: The wording should relate to issues and, more generally, requests (as in feature requests), but _not_ questions. The issue tracker is not a place for questions.

**Review by jonasmalacofilho — COMMENTED** [review]
This is something I've been meaning to do for a while. Thanks for looking into it!

Besides the specific comments, please also mind the capitalization of "liquidctl" (lowercase, unless at the start of a sentence, but try to avoid that) and of labels in general (I don't have a strong preference, just be consistent regarding whether only the first or all words are title cased).

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (gravityfargo) [reply .github/ISSUE_TEMPLATE/bug_report.yml:4]: I moved the entries around to reflect your comment and the current template.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -58,4 +43,37 @@ body:
...
+    validations:
+      required: true
+  - type: input
+    id: info-version
+    attributes:
+      label: Version of liquidctl
+      description: Output of `liquidctl --version` If the install failed, indicate the version you tried to install.
+      placeholder: e.g. liquidctl v1.9.0 (Linux-5.16.16-arch1-1-x86_64-with-glibc2.35) 
+    validations:
+      required: true
```
Comment (gravityfargo) [inline .github/ISSUE_TEMPLATE/bug_report.yml:78]: I couldn't think of a more concise way of putting this.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 5 above)
Comment (gravityfargo) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I moved this code around. I made a new comment highlighting the changes.

File: .github/ISSUE_TEMPLATE/help-request.yml (hunk 8 above)
Comment (gravityfargo) [reply .github/ISSUE_TEMPLATE/help-request.yml]: I deleted this form entirely and added a link to the discussions where I had put the discord link

**Comment by gravityfargo** [comment]
Updated live examples in my Repo:

Replaced Discord with Discussion [template picker page](https://github.com/gravityfargo/liquidctl/issues/new/choose). 

[bug report form](https://github.com/gravityfargo/liquidctl/issues/new?assignees=&labels=bug&template=bug_report.yml)
[example issue](https://github.com/gravityfargo/liquidctl/issues/4)

[new device form](https://github.com/gravityfargo/liquidctl/issues/new?assignees=&labels=new+device&template=new-device.yml)
[example issue](https://github.com/gravityfargo/liquidctl/issues/5)

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 9 above)
Comment (jonasmalacofilho) [reply .github/ISSUE_TEMPLATE/bug_report.yml:78]: It's fine!

**Comment by jonasmalacofilho** [comment]
I might take another look later, and eventually tweak a thing or two (e.g. which fields are required). But it's certainly good enough to merge, and an improvement over our current templates.

Thanks!

**Pull request merged**